mailgun/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class HttpResponseException(RuntimeError):
```

Comment:
Why is this called `HttpResponseException`? It seems to always encapsulate JSON responses (to mailgun), and I can't see any references to the new `HttpResponse` imports in route_handlers.py or tests.py. Please educamate me.